packages/mobile/capacitor/plugins/secure-filesystem-access/ios/Plugin/Plugin.m

```diff
@@ -8,5 +8,5 @@
     CAP_PLUGIN_METHOD(removeProfileFolder, CAPPluginReturnPromise);
     CAP_PLUGIN_METHOD(renameProfileFolder, CAPPluginReturnPromise);
     CAP_PLUGIN_METHOD(listProfileFolders, CAPPluginReturnPromise);
-    CAP_PLUGIN_METHOD(saveRecoveryKit, CAPPluginReturnPromise);
+    CAP_PLUGIN_METHOD(saveTextFile, CAPPluginReturnPromise);
```

Comment:
I'm a little further. The old method name for saveRecoveryKit was still registered so I added the new one here. But now the app crashes here (line 131):

![image](https://user-images.githubusercontent.com/6748220/185882280-2e0ee3f2-4ba3-4e03-ab14-30c60f3b6a56.png)

@amadeu2 I sent you the error log on slack

Reply:
Nice catch! 💯💪